.travis.yml

```diff
@@ -6,9 +6,12 @@ install:
 
 script:
   - tox -e py27
+  - (cd core && tox -e py27)
   - tox -e py34
+  - (cd core && tox -e py34)
   - tox -e lint
   - tox -e cover
+  - (cd core && tox -e cover)
```

Comment:
I agree. I intend to write a script to manage which tests get run (and what directory they get run from). Along the lines of #2277.

This super-hack was just a quick and dirty way to get Travis to run those tests.